lib/project_types/script/layers/domain/extension_point.rb

```diff
@@ -9,24 +9,74 @@ class ExtensionPoint
         def initialize(type, config)
           @type = type
           @deprecated = config["deprecated"] || false
-          @sdks = {
-            ts: ExtensionPointAssemblyScriptSDK.new(config["assemblyscript"]),
-          }
+          @sdks = ExtensionPointSDKs.new(config)
         end
 
         def deprecated?
           @deprecated
         end
       end
 
-      class ExtensionPointAssemblyScriptSDK
-        attr_reader :package, :version, :sdk_version, :toolchain_version
+      class ExtensionPointSDKs
+        def initialize(config)
+          @config = config
+        end
+
+        def all
+          [assemblyscript, rust].compact
+        end
+
+        def assemblyscript
+          @assemblyscript ||= new_sdk(ExtensionPointAssemblyScriptSDK, @config['assemblyscript'])
```

Comment:
Nitpick: instead of hardcoding `assemblyscript`, should we call `ExtensionPointAssemblyScriptSDK.language`?